### PR TITLE
Client QA items

### PR DESCRIPTION
**Pull request by nahbee10**
### Description

Fixed these items from the client QA board on Notion([link](https://www.notion.so/garden3d/142131fea2c78138bc87eab60d594492?v=142131fea2c7814b9724000cdb974c5f&pvs=4), all the items under the 'in progress' column)
- Change “Ecosystem” to “Protocol” in footer
- Line height of menu items in side bar is too tall
- Header displaying weird scroll bar 
- Grey scrollboxes around side bar
- Link text isn’t wrapping to the next line properly
- Add hover states to featured CTAs
- Remove ‘Connect with us’ section from Home page 
- In article pages, collapse parent section in side bar by default
- Suggested search dropdown extends past search bar
- Slightly increase line height of text in articles

### Type(s) of changes

- [x] Bug fix
- [ ] New feature
- [x] Update to an existing feature

### Motivation for PR

Client QA items under the 'in progress' column([link](https://www.notion.so/garden3d/142131fea2c78138bc87eab60d594492?v=142131fea2c7814b9724000cdb974c5f&pvs=4))

### How Has This Been Tested?

tested on local and production preview

**HOW TO PREVIEW:
Step 1. Login to Uniswap Labs Zendesk 

Step 2. Open this link(https://support.uniswap.org/theming/theme/65eaf0e7-4b24-40f6-bdf1-34d5c46d85a2) to activate the staging theme(will not affect the production theme)

Step 3. Open this link(https://support.uniswap.org/hc/en-us) to view the staging site without iframe nest

### CODE REVIEW

**Review by mokaymokay — CHANGES_REQUESTED** [review]
See comments: https://www.loom.com/share/6e251c479cd84ff0b8b427b0b58a1887

**Comment by nahbee10** [comment]
> See comments: 

@mokaymokay addressed the comments in the preview(use the same way above to preview it!)

1. fixed the font size of <li>
Screenshot 2024-12-10 at 8 35 02 PM

2. added bottom padding to the heading texts:
Screenshot 2024-12-10 at 8 09 58 PM

3. removed hover state from chevron icon on the right side of side nav items:
Screenshot 2024-12-10 at 8 34 39 PM

4. applied shade to the right side of overflowing breadcrumb([Figma design](https://www.figma.com/design/QkBERlCkBKBAWRqELCC8uq/%5BUniswap%5D-%F0%9F%8D%84-Spore-Library-w%2F-Web?m=auto&node-id=5083-15456))
Screenshot 2024-12-10 at 8 38 24 PM

5. align autocomplete search result window to the search text field width:
Screenshot 2024-12-10 at 8 23 33 PM

**Review by mokaymokay — APPROVED** [review]
Noticing one more thing!

If I'm on a category page, looks like the parent is highlighted but the category isn't highlighted on mobile/tablet nav. 
Screenshot 2024-12-10 at 2 43 13 PM

See intended behavior on desktop
Screenshot 2024-12-10 at 2 44 02 PM

File: src/modules/side-nav/MobileMenuNavModule.tsx [hunk 1]
```diff
@@ -96,6 +96,7 @@ export const MobileMenuNav: FC<Props> = ({ sideNavData, navState }) => {
           <ul className="accordion-body">
             {selectedCategory &&
               selectedCategory.sections.map((section) => {
+                console.log(section, navState.category, selectedCategory.id);
```
Comment (mokaymokay) [inline src/modules/side-nav/MobileMenuNavModule.tsx:99]: remove?

**Review by mokaymokay — APPROVED** [review]

**Pull request merged**